pkg/platform/aks/aks.go

```diff
@@ -76,7 +76,7 @@ const (
 	subscriptionIDEnv = "LOKOMOTIVE_AKS_SUBSCRIPTION_ID"
 	tenantIDEnv       = "LOKOMOTIVE_AKS_TENANT_ID"
 
-	kubernetesVersion = "1.16.10"
```

Comment:
is that first available jump ? No other minor version before `1.17.9` ?

Reply:
There are following version available for updates: `1.16.13, 1.17.7, 1.17.9`, but I think we should just go to the latest available.

Reply:
ok.